README.md

```diff
@@ -125,6 +113,40 @@ Repo of the Python module JIWER: https://pypi.org/project/jiwer/
 It computes the minimum-edit distance between the ground-truth sentence and the hypothesis sentence of a speech-to-text API.
 The minimum-edit distance is calculated using the python C module python-Levenshtein.
 
+## Execution
+
+```
+python analyze.py --config_file config.ini --log_level DEBUG
+```
+
+# Experimenting
+Use the `experiment.py` script to execute a series of Transcription/Analyze experiments to optimize SpeechToText parameters. 
+
+## Setup
+
```

Comment:
Sorry if i missed it but do you have the `-c` and `-ll` flag options documented somewhere?

Reply:
added a separate section to explain since they are common parameters and then linked to it

Reply:
@tnixa Please confirm this is resolved to your satisfaction and resolve conversation